.github/workflows/test.yaml

```diff
@@ -0,0 +1,18 @@
+name: Check Markdown and Examples
+on:
+  pull_request:
+    types: [opened, synchronize]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/setup-node@v1
+      - uses: actions/setup-python@v2.2.2
+      - uses: actions/checkout@v2
+      - run: pip install poetry
+      - run: poetry install
+      - run: npm install
```

Comment:
use yarn.

Reply:
What's the easiest way to migrate? I haven't used Yarn for years. Also, it looks like [recent Yarn is not supported by Dependabot](https://github.com/dependabot/dependabot-core/issues/1297), which might be a blocker.

Reply:
we use dependabot on all of our yarn repos, we are in the process of getting everyone moved from npm to yarn.

```
rm package-lock.json
yarn
git add yarn.lock
git commit -am 'chore: switch to yarn
```